spec/predicates/spdx.md

```diff
@@ -1,47 +1,69 @@
 # Predicate type: SPDX
 
-Type URI: (tentative) https://spdx.dev/Document
+Type URI: https://spdx.dev/Document
```

Comment:
Should we somehow indicate that the Type URI MUST include a version?
```suggestion
Type URI: https://spdx.dev/Document/v${SPDX_MAJOR}.${SPDX_MINOR}
```

Reply:
I like this idea, though we haven't required this in other predicate definitions either. 

This may be a question beyond the scope of this PR, but it seems like we might want to be clearer in our documentation for predicateTypes.